src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -119,6 +129,35 @@ CreateSpaceProcessor::pickHosts(PartitionID partId,
     return pickedHosts;
 }
 
+int32_t CreateSpaceProcessor::GetPartsNumbers() {
+    int32_t count = 0;
+    auto prefix = MetaServiceUtils::spacePrefix();
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto ret = kvstore_->prefix(kDefaultSpaceId, kDefaultPartId, prefix, &iter);
+    if (ret != kvstore::ResultCode::SUCCEEDED) {
+        return -1;
```

Comment:
I don't know what it means to return -1 here. the caller does not process it. 
If here failed, is means the rocksdb may be abnormal, and we should return and don’t continue.

Reply:
If return -1, we should throw a exception here?

Reply:
fix this